README.md

```diff
@@ -11,7 +11,7 @@ Math magician is a website for all fans of mathematics. It is a single Page App
 - [x] Project 1: Setup
```

Comment:
The [Microverse readme template](https://github.com/microverseinc/readme-template) is a requirement for most projects at Microverse, according to the [Professional Repo Basic Rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) general requirements.

Look:

![image](https://user-images.githubusercontent.com/57421823/158829503-2c877c5c-bf62-4458-a368-f383743f96aa.png)


- [ ] On your readme, please kindly implement the [Microverse readme template](https://github.com/microverseinc/readme-template).  All the sections are required except for "live demo" and "tests" if not required.

 Hint:  You can include the section that you already have written.